resources/views/shop/detail.blade.php

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>お店の登録フォーム</title>
+
+</head>
+
+<body>
+    <h1>{{ $shop->name }}</h1>
+    <p>住所：{{ $shop->address }}</p>
+@auth
```

Comment:
Laravel directive for redirecting to login page if the user has not logged in yet